src/components/MapView/SimpleDropdown/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { FormControl, MenuItem, Select } from '@material-ui/core';
+import React from 'react';
+
+type OptionValue = number | string;
+type OptionLabel = string;
+type DropdownOption = [OptionValue, OptionLabel];
+
+interface SimpleDropdownProps {
+  options: DropdownOption[];
+  value: OptionValue;
+  // want to do (val: OptionValue)
+  // but this causes type conflicts when a user knows that val will be an enum
+  onChange: (val: any) => void;
```

Comment:
I refactored OptionValue to be a constrained generic instead of a static type and so we can just remove any here and use the React.ChangeEvent type assigned by default.   But I did replace the onChange param type from any to OptionValue.  Here's the commit:

https://github.com/WFP-VAM/prism-frontend/pull/431/commits/9c857a165c8f186868f94e89a1e3df54fc091279

I'm going to leave this one open for your review, because constrained generics are complicated.  Let me know if you have any feedback or questions.